adafruit_simplemath.py

```diff
@@ -14,23 +13,47 @@
 Implementation Notes
 --------------------
 
-**Hardware:**
-
-.. todo:: Add links to any specific hardware product page(s), or category page(s). Use unordered list & hyperlink rST
-   inline format: "* `Link Text <url>`_"
-
 **Software and Dependencies:**
 
 * Adafruit CircuitPython firmware for the supported boards:
   https://github.com/adafruit/circuitpython/releases
-
-.. todo:: Uncomment or remove the Bus Device and/or the Register library dependencies based on the library's use of either.
-
-# * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
-# * Adafruit's Register library: https://github.com/adafruit/Adafruit_CircuitPython_Register
 """
 
-# imports
-
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_SimpleMath.git"
+
+
+def map_range(x, in_min, in_max, out_min, out_max):
+    """
+    Maps a number from one range to another. Somewhat similar to the Arduino ``map()`` function, but
+    returns a floating point result, and constrains the output value to be between ``out_min`` and ``out_max``.
+
+    :return: Returns value mapped to new range
+    :rtype: float
+    """
```

Comment:
@dhalbert Is it worth documenting the parameters with `:param: etc`?

Reply:
I can do that.